src/client/resources/conversions.ts

```diff
@@ -472,12 +472,15 @@ export function updateMigPlanFromValues(
 
       return updatedPV;
     });
-    updatedSpec.persistentVolumes = planValues.persistentVolumes;
+    updatedSpec.persistentVolumes = _.union(
```

Comment:
It looks like `_.union` will use referential equality to compare the objects, so if there is a PV that already appears in form state and is also in updatedSpec (a new copy of it, not the same reference), the PV would get included twice right? We might want to instead use [`_.unionBy()`](https://lodash.com/docs/4.17.15#unionBy) and give it a function for the second param that reduces each PV to a unique identifier

Reply:
Updated to include the iterator for the unique ID. @mturley